src/translib/transformer/xfmr_vrf.go

```diff
@@ -176,6 +180,8 @@ func init() {
         XlateFuncBind("DbToYang_network_instance_table_key_xfmr", DbToYang_network_instance_table_key_xfmr)
         XlateFuncBind("YangToDb_network_instance_enabled_field_xfmr", YangToDb_network_instance_enabled_field_xfmr)
         XlateFuncBind("DbToYang_network_instance_enabled_field_xfmr", DbToYang_network_instance_enabled_field_xfmr)
+        XlateFuncBind("YangToDb_network_instance_name_key_xfmr", YangToDb_network_instance_name_key_xfmr)
```

Comment:
This will return empty Map. This has to be defined at the key level. The NULL map has to be returned at the config level. That is defined in YangToDb_network_instance_name_field_xfmr